core/src/main/java/org/apache/hadoop/shaded/net/jpountz/lz4/LZ4Compressor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.shaded.net.jpountz.lz4;
+
+/**
+ * A temporary workaround for SPARK-36669. We should remove this after Hadoop 3.3.2 release
```

Comment:
In addition to `We should remove this after Hadoop 3.3.2 release`, we had better have an IDed TODO officially. If we already have could you add it here, please?

Reply:
I've not created one, let me create it and add it here. Thanks.

Reply:
Added `TODO(SPARK-36679)`.